package/MDAnalysis/core/universe.py

```diff
@@ -326,7 +326,12 @@ def __init__(self, *args, **kwargs):
             if not coordinatefile:
                 coordinatefile = None
             # parse transformations
-            self._transforms = kwargs.pop('transformations', [])
+            self._transforms = []
+            trans_arg = kwargs.pop('transformations', None)
+            if isinstance(trans_arg, list):
+                self._transforms = trans_arg
+            elif trans_arg:
+                self._transforms.append(kwargs.pop('transformations'))
```

Comment:
The argument is already popped out of the dict. It cannot be popped again. If this did not break the tests, it means you are missing a test. Did you test you can pass a callable to the universe keyword argument?

Reply:
You are right. I was unsure if I would do the checking or just force the args to be lists, and when I rolled back the changes I messed up.

Reply:
From what I see, the test is still missing.

Reply:
Should I add the test in ``test_universe.py`?

Reply:
Sounds like a good place indeed.

Reply:
I added tests for both cases: callable and workflow list

Reply:
You did something to the file, it appears as every line changed. Something related to windows end of lines maybe?